fuel-tx/src/tests/offset.rs

```diff
@@ -429,7 +434,6 @@ fn tx_offset_script() {
 
 #[test]
 fn tx_offset_mint() {
```

Comment:
Can you explain to me what this is testing? I'm not familiar with the concept of "offsets" in this context.

Reply:
The test verifies that [offset getters](https://github.com/FuelLabs/fuel-vm/pull/593/files#diff-e0db6cbfd4a6b1d090f2fd45858593b8c51e6adecb94b4ffbddad14065855df9) for each field are correct.

Later, we can use those offsets to extract values directly from the canonical byte representation of the transaction instead of deserializing the whole object(it can be quite expensive).

For example, we already use it for `Script` transactions. During the execution of the script, its fields are serialized into the VM memory, and the programmer of the Sway contract has access to them.